src/api/app/models/distribution.rb

```diff
@@ -38,8 +38,7 @@ def self.all_including_remotes
     list = Distribution.all_as_hash
     repositories = list.map{ |d| d['reponame'] }
 
-    remote_projects = Project.where("NOT ISNULL(projects.remoteurl)")
```

Comment:
Can you remove this?

Won't it fails at https://github.com/openSUSE/open-build-service/pull/2449/files#diff-85e46cbaea9243ab1730d9fc6779dca6L45 with `undefined method '+' for nil:NilClass`?